remove unsupported verbs

jmclaughlin-bw opened the pull request:
They aren't coming soon :)

dtolb commented:
Had some conflicts due to the way I set this up.

However, external to that, be sure to update the:

* [Summary](https://github.com/Bandwidth/ap-docs/blob/bxml-updates/SUMMARY.md)
* [Root BXML](https://github.com/Bandwidth/ap-docs/blob/bxml-updates/bxml/bxml.md)

A good way to find if there are links is to search the entire project for the file name you deleted. 

I've updated the conflicts and removed the links.

jmclaughlin-bw commented:
Thanks :)



[image: email-signature-logo.jpg]

Jennifer McLaughlin

*Director, Product Management - Voice APIs*

900 Main Campus Drive, Suite 400

Raleigh, NC 27606

Bandwidth
<http://www.bandwidth.com>m 919.368.5336

e j <email@bandwidth.com>mclaughlin@bandwidth.com

On Mon, Mar 13, 2017 at 11:09 AM, Daniel Tolbert <notifications@github.com>
wrote:

> Had some conflicts due to the way I set this up.
>
> However, external to that, be sure to update the:
>
>    - Summary
>    <https://github.com/Bandwidth/ap-docs/blob/bxml-updates/SUMMARY.md>
>    - Root BXML
>    <https://github.com/Bandwidth/ap-docs/blob/bxml-updates/bxml/bxml.md>
>
> A good way to find if there are links is to search the entire project for
> the file name you deleted.
>
> I've updated the conflicts and removed the links.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Bandwidth/ap-docs/pull/19#issuecomment-286136255>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/APjV4fCzUwxGELYimAxi-GPWhXiZlvYhks5rlVwlgaJpZM4MZZyJ>
> .
>
